backend/src/test/java/com/example/backend/service/SightServiceTest.java

```diff
@@ -64,8 +63,34 @@ void getSightById_whenSightNotExists_throwException() {
         when(sightRepo.findById("1")).thenReturn( Optional.empty());
 
         // WHEN & THEN
-        //verify(sightRepo).findById("1");
         assertThrows(NoSuchElementException.class, () -> sightService.getSightById("1"));
+        verify(sightRepo).findById("1");
+    }
+
+
+    @Test
+    void addNewSight_whenSightDTO () {
+        // GIVEN
+        SightDTO sightDTO = new SightDTO("AAA", "BBB", "CCC", "DDD", "EEE", "HHH", "TTT");
+        when(idService.generateID()).thenReturn("1");
+        when(sightRepo.save(any())).thenReturn(Sight.builder()
+                .id("1")
+                .name(sightDTO.getName())
+                .image(sightDTO.getImage())
+                .address(sightDTO.getAddress())
+                .website(sightDTO.getWebsite())
+                .time(sightDTO.getTime())
+                .description(sightDTO.getDescription())
+                .location(sightDTO.getLocation())
+                .build());
```

Comment:
```suggestion
new Sight("1","AAA", "BBB", "CCC", "DDD", "EEE", "HHH", "TTT")
```
Ich würde keinen Builder im Test benutzen, tue am besten einfach ein neues Sight erstellen.